index.ts

```diff
@@ -1,18 +1,24 @@
 import { Database } from '@hocuspocus/extension-database';
 import { Logger } from '@hocuspocus/extension-logger';
 import { Server } from '@hocuspocus/server';
-import { Storage } from '@google-cloud/storage';
+import { Bucket, Storage } from '@google-cloud/storage';
 import { utils } from './stream-helper'
 import cors from 'cors';
 import express from 'express';
 import expressWebsockets from 'express-ws';
 import ShortUniqueId from 'short-unique-id';
+import fileStore from './fileStore';
 
 const uid = new ShortUniqueId({ length: 20 });
 
 const serverPort = parseInt(process.env.SERVER_PORT || '8080');
-const storageClient = new Storage();
-const storageBucket = storageClient.bucket(process.env.BUCKET_NAME || 'athene-diagram-files');
+let storageBucket: Bucket | typeof fileStore;
+if (process.env.BUCKET_NAME) {
```

Comment:
I don't link this Env variable:

1. You initialioze the gcp store, even if the bucket is not set.
2. I think there should be a better env name like STORAGE_TYPE or so and then STORAGE_GCP_BUCKET_NAME

Reply:
Makes sense, let me incorporate these changes.

Reply:
1. not able to understand this point, is it initializing due to import? I am only creating the instance of Storage bucket if the condition is true?
2. have included these points, server now expects STORAGE_TYPE and STORAGE_GCP_BUCKET_NAME, if both of these are not set (as in your use case) it will default to gcp and athene-diagram-files bucket name.

Reply:
You have solved the first point now :)